src/Pelagos/Entity/DOI.php

```diff
@@ -153,6 +153,6 @@ public function setPublicDate(\DateTime $publicDate)
      */
     public function __toString()
     {
-        return (string) 'doi:' . $this->doi;
```

Comment:
doi is part of the display value. As recommended by DOI org. That's what makes this function unique, otherwise it returned the same thing as getDoi()

Reply:
Yeah, I can change it back. We should probably use this at landing and other display pages, I assumed that it wasn't necessary to have the 'doi:'